integration-tests/oidc-code-flow/src/test/java/io/quarkus/it/keycloak/CodeFlowTest.java

```diff
@@ -1282,6 +1282,15 @@ public void testDefaultSessionManagerIdRefreshSplitTokens() throws IOException,
             loginForm.getInputByName("password").setValueAttribute("alice");
 
             page = loginForm.getInputByName("login").click();
+
```

Comment:
@pedroigor This is what I had to do to fix one of the CodeFlowTest, DevService for Keycloak is supporting this test. I've checked the forums, looks like there is a user profile which requires these extra fields is enabled by default. 

If I start `quarkus-quickstarts/security-openid-connect-web-authentication-quickstart` with `mvn quarkus:dev`, and access `http://localhost:8080/tokens` from the browser then I don't see these extra required fields.

CodeFlowTest itself has 40+ tests but only 22 of them fail if these fields are not submitted, so it seems like Keycloak is choosing nearly randomly when to request them.

May be I can toggle it back to the username and password only somehow in the the start command ?

Reply:
@famod We've discussed with Pedro how to handle it. In the end I've actually tried to fix all the tests the way I did with this one, but I've found that this very test sometimes does not require the extra 3 attributes be submitted. So an option of updating the `UPConfig` will need to be explored as also suggested by Pedro